client/src/theme/styles/components/global/notifications/notification.js

```diff
@@ -35,6 +35,10 @@ export default `
       @starting-style {
         opacity: 0;
       }
+
+      &.notification--error {
+        background-color: var(--color-notification-error-light);
+      }
```

Comment:
I went with the existing notification styles with an error style for now. Open to ideas for other ways to style/display this.